frontend/src/use-cases/home/home.js

```diff
@@ -1,23 +1,19 @@
-import React, { useState, useEffect } from "react";
+import React, { useContext } from "react";
 
-import "./home.css";
-import Prompt from "./views/Prompt/prompt.view";
+import Prompt from "./views/Prompt";
 import SuggestionBoard from "./views/SuggestionBoard/SuggestionBoard";
-import { checkLogin } from "../../services/data.service";
+import UserContext from "common/context/user-context";
+import { HomeWrapper, Divider } from "./home.style";
```

Comment:
should be Home.style

Reply:
All files accept App and SuggestionBoard start with loser case

Reply:
Why?

Reply:
That doesn't follow cthit/style-guides